src/utility_functions.jl

```diff
@@ -152,6 +152,31 @@ end
     r == 3 && return rotate_matrix_indices_270(i,j,s)
 end
 
+"""
+Downloads input files as required by SpeedyWeather.jl
+"""
+function download_input_data(input_data_path)
+    input_data_url = "https://github.com/dmey/speedy-weather-input-data/archive/refs/heads/main.zip"
```

Comment:
Yeah, we shouldn't hardcode the url. But I'm not sure what the best way is to provide a nice interface, which remains compatible or at least provides some control between input data version and SpeedyWeather.jl version. I like the idea to just create a download some netcdf files and create a folder once. But not sure what undesired consequences of that are compared to having a registered package with version and we can then adapt and adjust version compatibility in SpeedyWeather.jl.